diagnostics/diagnostic.go

```diff
@@ -73,9 +74,18 @@ func interfaceToJSONString(i interface{}) string {
 	return string(b)
 }*/
 
+func (d *DiagnosticClient) findNodeDisk() string {
+	dirPath := d.node.Backend().DataDir()
```

Comment:
high chance that `Dirs.Chaindata` and `Dirs.SnapDomain` is monted/symlinked to another drive